core/src/main/java/com/linecorp/armeria/client/Endpoint.java

```diff
@@ -643,7 +643,7 @@ private static void validateWeight(int weight) {
     }
 
     @Override
-    public boolean equals(Object obj) {
+    public boolean equals(@Nullable Object obj) {
```

Comment:
Question: do we decide to put `@Nullable` here?

Reply:
Yeah, all `equals()` parameters are `@Nullable`, because `null` is accepted indeed.